src/main/proto/wfa/measurement/internal/kingdom/exchange.proto

```diff
@@ -0,0 +1,31 @@
+// Copyright 2021 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package wfa.measurement.internal.kingdom;
+
+import "google/type/date.proto";
+import "src/main/proto/wfa/measurement/internal/kingdom/exchange_details.proto";
+
+option java_package = "org.wfanet.measurement.internal.kingdom";
+option java_multiple_files = true;
+
+// Proto representation of the internal Exchange entity type.
```

Comment:
Can you document what an Exchange is? 

For example:

An Exchange is a record of a particular invocation of a panelist data exchange between a panel provide and an EDP.